java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -1903,9 +1903,33 @@ public ColumnVector asTimestamp(DType timestampType, String format) {
    * @return A new vector allocated on the GPU.
    */
   public ColumnVector asStrings() {
+    if (type.isTimestamp()) {
```

Comment:
I am not 100% sure that is what we want as the default.  Each timestamp has a different precision, and we probably want to print up to that precision.  So for TIMESTAMP_DAYS I would like to see "%Y-%m-%d", for TIMESTAMP_SECONDS this is great, for the other higher resolution timestamps I would prefer to see a `".%f"` at the end.  ANd it would be good to document and test this.

Reply:
We should also put the same warning about negative timestamps not producing correct results here.